Source/FileSystem/Json.h

```diff
@@ -15,6 +15,7 @@ class Json
 
 	bool fromBuffer(char*& buffer);
 	void toBuffer(rapidjson::StringBuffer& buffer);
+	void getVectorNames(std::vector<const char*> &vec);
```

Comment:
Why have it as an output parameter and not as a return type? Also, "get" should be capitalized.
```suggestion
	std::vector<std::string> GetVectorNames();
```